libera/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'groovy'
+apply plugin: 'application'
+apply plugin: 'maven'
+apply plugin: 'build-dashboard'
+
+group = 'com.connexta.libera'
+version = '1.0.0-SNAPSHOT'
+
+mainClassName = 'com.connexta.libera.Libera'
+
+dependencies {
+    compile 'org.codehaus.groovy:groovy-all:2.4.4'
+    compile 'org.jyaml:jyaml:1.3'
+    compile 'commons-cli:commons-cli:1.3.1'
+    compile 'com.connexta.libero:libero:1.0.0-SNAPSHOT'
+}
+
+test {
```

Comment:
this could probably be moved to the root build script under the `subproejects` section since it applies to all projects